tests/integration/test_logcollector/test_location/test_exclude.py

```diff
@@ -0,0 +1,189 @@
+               os.path.join(temp_dir, 'wazuh-testing', 'test2.txt')],
+     'log_format': 'syslog', 'exclude':os.path.join(temp_dir, 'wazuh-testing', 'test*'),
+     'expected_matches': [os.path.join(temp_dir, 'wazuh-testing', '1test1.txt'),
+                          os.path.join(temp_dir, 'wazuh-testing', '1test1.log'),
+                          os.path.join(temp_dir, 'wazuh-testing', '2test2.log')],
+     'description': 'Testing right and left wildcard, right exclude'},
+    {'location': os.path.join(temp_dir, 'wazuh-testing', 'test*'),
+     'files': [os.path.join(temp_dir, 'wazuh-testing', 'test1.txt'),
+               os.path.join(temp_dir, 'wazuh-testing', 'test1.log'),
+               os.path.join(temp_dir, 'wazuh-testing', 'test2.log')],
+     'log_format': 'syslog', 'exclude': os.path.join(temp_dir, 'wazuh-testing', '*test*'),
+     'expected_matches': ['none'],
+     'description': 'Testing right wildcard, right and left exclude'},
+    {'location': os.path.join(temp_dir, 'wazuh-testing', '*test.txt'),
+     'files': [os.path.join(temp_dir, 'wazuh-testing', '1test.txt'),
+               os.path.join(temp_dir, 'wazuh-testing', '2test.txt')],
+     'log_format': 'syslog', 'exclude': os.path.join(temp_dir, 'wazuh-testing', '*test*'),
+     'expected_matches': ['none'],
+     'description': 'Testing left wildcard, right and left exclude'},
+    {'location': os.path.join(temp_dir, 'wazuh-testing', '*test*'),
+     'files': [os.path.join(temp_dir, 'wazuh-testing', '1test1.txt'),
+               os.path.join(temp_dir, 'wazuh-testing', '1test1.log'),
+               os.path.join(temp_dir, 'wazuh-testing', '2test2.log'),
+               os.path.join(temp_dir, 'wazuh-testing', 'test2.txt')],
+     'log_format': 'syslog', 'exclude': os.path.join(temp_dir, 'wazuh-testing', '*test*'),
+     'expected_matches': ['none'],
+     'description': 'Testing right and left wildcard, right and left exclude'},
+    {'location': os.path.join(temp_dir, 'wazuh-testing', '*'),
+     'files': [os.path.join(temp_dir, 'wazuh-testing', '1test1.txt'),
+               os.path.join(temp_dir, 'wazuh-testing', '1test1.log'),
+               os.path.join(temp_dir, 'wazuh-testing', '2test2.log'),
+               os.path.join(temp_dir, 'wazuh-testing', 'test2.txt')],
+     'log_format': 'syslog', 'exclude': os.path.join(temp_dir, 'wazuh-testing', '*'),
+     'expected_matches': ['none'],
+     'description': 'Testing wildcard location'}
+
+]
+
+# Configuration data
+configurations = load_wazuh_configurations(configurations_path, __name__, params=parameters, metadata=metadata)
+configuration_ids = [f"{x['LOCATION']}_{x['LOG_FORMAT']}" for x in parameters]
+wazuh_log_monitor = FileMonitor(LOG_FILE_PATH)
+
+
+# Fixtures
+@pytest.fixture(scope="module")
+def create_directory():
+    """Create expected directories."""
+    os.makedirs(os.path.join(temp_dir, 'wazuh-testing'), exist_ok=True)
+    yield
+    rmtree(os.path.join(temp_dir, 'wazuh-testing'), ignore_errors=True)

```

Comment:
```suggestion
    os.makedirs(os.path.join(temp_dir, 'wazuh-testing'), exist_ok=True)
    
    yield
    
    rmtree(os.path.join(temp_dir, 'wazuh-testing'), ignore_errors=True)

```

Reply:
Fixed in 675fe8071d37aabfec50859993e777238e0f3f45